src/content/1.7/modules/core_updates/1.7.7.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Changes in PrestaShop 1.7.7
+menuTitle: Changes in 1.7.7
+---
+
+# Notable changes in PrestaShop 1.7.7
+
+## Order view page layout modifications
+
+In this version the order page has been migrated with Symfony. We used this migration as an opportunity to improve the page layout to meet the merchants' needs.
+You can see a representation of these new hooks in the page here:
```

Comment:
You need to leave an empty line between paragraphs if you intend to keep these two lines separate

```suggestion
In this version the order page has been migrated with Symfony. We used this migration as an opportunity to improve the page layout to meet the merchants' needs.

You can see a representation of these new hooks in the page here:
```

Reply:
No I didn't plan on separating them, it was for readability in the file itself. You prefer to leave it all in a single line then?

Reply:
It's okay